data/src/main/java/io/gnosis/data/db/HeimdallDatabase.kt

```diff
@@ -3,28 +3,33 @@ package io.gnosis.data.db
 import androidx.room.Database
 import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
-import io.gnosis.data.db.daos.Erc20TokenDao
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
 import io.gnosis.data.db.daos.SafeDao
-import io.gnosis.data.models.Erc20Token
 import io.gnosis.data.models.Safe
 import io.gnosis.data.models.SafeMetaData
 
 @Database(
     entities = [
         Safe::class,
-        SafeMetaData::class,
-        Erc20Token::class
+        SafeMetaData::class
     ], version = HeimdallDatabase.LATEST_DB_VERSION
 )
 @TypeConverters(SolidityAddressConverter::class)
 abstract class HeimdallDatabase : RoomDatabase() {
 
     abstract fun safeDao(): SafeDao
 
-    abstract fun erc20TokenDao(): Erc20TokenDao
-
     companion object {
         const val DB_NAME = "safe_db"
-        const val LATEST_DB_VERSION = 1
+        const val LATEST_DB_VERSION = 2
+
+        val MIGRATION_1_2 = object : Migration(1, 2) {
```

Comment:
Can we have a migration from 2 -> 1. So we can easily keep our safes when going back a db version. I had to delete all my safes because the app would crash, when I installed a build from master (same versionId.).

Reply:
it's very unusual. Migrations are applied in an ascending order. I don't think it is possible